docs/solution-design.md

```diff
@@ -6,6 +6,50 @@ While there are many ways to achieve the goal we are looking for, for the purpos
 
 ## Components used and their roles
 
-## Message Flow
+The following components will be used as part of the solution:
+  
+- Cosmos DB Account
+- Cosmos DB Containers
+  - On Hand (onhand)
+  - Shipments (shipments)
+  - Master Data Container (mdc)
+- Azure Functions
+  - Change Feed Listeners
```

Comment:
Might be worth it to enumerate what these will be (On Hand and Shipments) to clarify there is no change feed listener for Master Data

Reply:
Added text in Azure Functions to make it clear that change feed listeners are only on the On Hand and Shipment CosmosDB containers.